core/src/apps/management/set_permanent_passphrase.py

```
@@ -0,0 +1,73 @@
+    # including its header (which would otherwise reveal the old length).
+    storage_common.compact()
+
+    # Clear cached secrets so the next operation derives from the new root.
+    ctx = context.get_context()
+    ctx.cache.delete(APP_COMMON_SEED)
+    try:
+        from storage.cache_common import APP_COMMON_SEED_WITHOUT_PASSPHRASE
+
+        ctx.cache.delete(APP_COMMON_SEED_WITHOUT_PASSPHRASE)
+    except Exception:
+        pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)